NEXT_CHANGELOG.md

```diff
@@ -26,6 +26,22 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 # [x.x.x] (unreleased) - 2022-mm-dd
 
 ## ❗ BREAKING ❗
+
+### Respect supergraph path for kubernetes deployment probes [PR #1795](https://github.com/apollographql/router/pull/1795)
```

Comment:
mis-merge or ?

Reply:
Oh yes, I tried to use the "update branch" from github but probably a bad idea :p

Reply:
Oh no, it's a bad copy/paste, my bad

Reply:
fixed